src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -501,6 +571,15 @@ internal abstract partial class BaseSimpleAssignmentExpression : AssignmentExpre
             base(OperationKind.SimpleAssignmentExpression, syntax, type, constantValue)
         {
         }
+        public override IEnumerable<IOperation> Children
```

Comment:
Why not implement this in the base Assignment class, and then let children override if they have different fields?